integration/qbft/tests/regular_test.go

```diff
@@ -0,0 +1,111 @@
+package tests
```

Comment:
@aaron-blox while this is still WIP, lets remove from the github actions to run integration tests until they are stable.
you can create a regex rule to run it if the branch starts/ends with integration-test

Reply:
resolved